cranelift/fuzzgen/src/lib.rs

```diff
@@ -48,7 +50,7 @@ impl fmt::Debug for TestCase {
 
         // Print only non default flags
         let default_flags = Flags::new(settings::builder());
-        for (default, flag) in default_flags.iter().zip(self.flags.iter()) {
+        for (default, flag) in default_flags.iter().zip(self.isa.flags().iter()) {
             assert_eq!(default.name, flag.name);
 
             if default.value_string() != flag.value_string() {
```

Comment:
I can't comment on the exact line (61), but can we print only the correct `target {}` line here?

Reply:
Yep! I'll duplicate the match in cranelift-compile.rs here.